spanner/spanner_snippets/spanner/pg_spanner_create_storing_index.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_postgresql_create_storing_index]
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	database "cloud.google.com/go/spanner/admin/database/apiv1"
+	adminpb "google.golang.org/genproto/googleapis/spanner/admin/database/v1"
+)
+
+// pgAddStoringIndex shows how to create 'STORING' indexes on a Spanner
+// PostgreSQL database. The PostgreSQL dialect uses INCLUDE keyword, as
+// opposed to the STORING keyword of Cloud Spanner.
+func pgAddStoringIndex(ctx context.Context, w io.Writer, db string) error {
+	// db := "projects/my-project/instances/my-instance/databases/my-database"
+	adminClient, err := database.NewDatabaseAdminClient(ctx)
+	if err != nil {
+		return err
```

Comment:
https://googlecloudplatform.github.io/samples-style-guide/#errors suggests adding some context to error returns (here and following) to clarify meaning-- that could be a print statement to w and/or a comment and/or adding to the error string returned from the service.